openstack/mrs/v2/clusters/requests.go

```diff
@@ -448,3 +462,16 @@ func Create(client *golangsdk.ServiceClient, opts CreateOptsBuilder) (r CreateRe
 	_, r.Err = client.Post(rootURL(client), b, &r.Body, reqOpt)
 	return
 }
+
+// CreatePrepaid is a method to create a new prePaid mapreduce cluster.
+func CreatePrepaid(client *golangsdk.ServiceClient, opts CreateOptsBuilder) (r CreatePrepaidResult) {
+	b, err := opts.ToClusterCreateMap()
+	if err != nil {
+		r.Err = err
+		return
+	}
+
+	reqOpt := &golangsdk.RequestOpts{OkCodes: []int{200}}
+	_, r.Err = client.Post(rootURL(client), b, &r.Body, reqOpt)
+	return
+}
```

Comment:
There is unnecessary to support a extra function.
Put the `OrderId` into the `Cluster` structure just okey.

Reply:
done